packages/components/tree-v2/src/tree.type.ts

```diff
@@ -0,0 +1,48 @@
+import type {
```

Comment:
We'd better rename it to `types.ts`.
It not only used for `tree.vue` but also for whole tree-v2